packages/react/fast-components-react-base/src/index.ts

```diff
@@ -86,6 +86,11 @@ import SliderTrackItem from "./slider-track-item";
 export { SliderTrackItem };
 export * from "./slider-track-item";
 
+// holding off on exporting this until we are confident of the implementation
```

Comment:
This is going to look strange in our changelog with the current title as it implies the stack panel is now available, can you change it to a chore instead and use `feat` once these lines are safe to uncomment?

Reply:
Agree with this - we can still use the control internally without exporting it but if it goes into the changelog as though it was "added" it may confuse folks thinking it's available for export. Likely best to shift this to a chore.